dlgr/griduniverse/experiment.py

```diff
@@ -875,19 +900,35 @@ def rank(self, color):
             return 1
 
 
-class Food(object):
-    """Food."""
+class Object(object):
+    """Objects."""
     def __init__(self, **kwargs):
         self.id = kwargs.get('id', uuid.uuid4())
+        self.object_id = kwargs.get('object_id', 1)
+        self.name = kwargs.get('name')
         self.position = kwargs.get('position', [0, 0])
+        self.calories = kwargs.get('calories', 0)
         self.color = kwargs.get('color')
+        self.spawn_rate = kwargs.get('spawn_rate', 0.1)
+        self.portable = kwargs.get('portable', False)
+        self.crossable = kwargs.get('crossable', False)
+        self.interactive = kwargs.get('interactive', False)
+        self.n_uses = kwargs.get('n_uses', 1)
         self.maturation_speed = kwargs.get('maturation_speed', 0.1)
         self.creation_timestamp = time.time()
 
     def serialize(self):
         return {
             "id": self.id,
+            "object_id": self.object_id,
+            "name": self.name,
```

Comment:
We definitely don't want to serialize any of these properties except for `id`, `object_id`, `position` and `maturity`.